docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -213,6 +220,20 @@ The following command operators are supported:
     Connection termination details may provide additional information about why the connection was
     terminated by Envoy for L4 reasons.
 
+%RESPONSE_HEADERS_BYTES%
+  HTTP
+    Response headers bytes.
```

Comment:
It's worth clarifying that the byte count is the uncompressed sum of header keys and values. The wire bytes might be fewer, e.g. if using HPACK.

Reply:
Done.